### ISSUE TITLE
Running way too fast

### ISSUE THREAD

**Issue by ZXNova** [07th-mod/ace-higurashi-attorney#1]
When I open up the game the music plays way too fast. I'm not sure what to do about this.

**Comment by drojf** [comment]
@DoctorDiablo I noticed this too when I was briefly testing it earlier, and upon checking the actual audio files the engine really is playing too fast. Here is the actual opening music file so you can play it:
[bgml018.zip](https://github.com/07th-mod/ace-higurashi-attorney/files/8841672/bgml018.zip)

After some testing I found the issue was that the music files have a sample rate of 24000hz, but the engine only supports more standard sample rates like 44100hz 22050hz. You can re-convert the files to WAV or a very high quality OGG/MP3 to fix this (I tested a resampled WAV to 44100hz and it fixed the problem), or see if there's a different version of the engine you can use.

----

side note: for distribution, you don't need to pack the the images and sound back into the arc.nsa, if you just put them next to the .exe the game will load them (like in the root folder you have `img`, `sound`, `0.txt`, `AceHigurashiAttorney-en.exe`)

**Comment by ZXNova** [comment]
There's also an issue with the game opening up in Fullscreen when I don't want it to. Is there a way to make it stop opening up in full screen as default? I didn't do this at first but now it keeps doing that everytime and it messes up things

**Comment by drojf** [comment]
Press the "f' key to go out of fullscreen. It looks the like game remembers this setting until you next change it, so if you accidentally pressed it, it can be a problem....

I also noticed that when you're in fullscreen mode, you can't quit the game with ALT-F4, the only way I could figure out to quit is to press 'f' to go windowed and exit, or force close via task manager

**Comment by DoctorDiablo** [comment]
I think I managed to convert the audio files and get things playing properly. Is it only the mp3s that needed to be changed or did it affect the wav files too?

**Comment by drojf** [comment]
I did some more testing, it looks like all the WAV files are probably fine (game accepts 22050 or 44100 for WAV files), although I didn't individually test all the WAV files, just one of each type.

Any MP3 file which is not 44100 hz needs to be fixed (highlighted in blue in the below image):

![image](https://user-images.githubusercontent.com/1249449/172507778-f51dc98a-190a-4b89-bd4f-2703917baa3f.png)

That is, only `45`, `bgm0042_sfx`, `fang`, and `h-haze` don't need to be converted, all others you need to convert (might as well convert all of them at this point).

**Comment by ZXNova** [comment]
Will something be done?

**Comment by DoctorDiablo** [comment]
@ZXNova Yeah, I've got the audio all converted, it just need to be tested to make sure this doesn't somehow break things.  If you'd like to try it out, here are the files: [FixAudioIssue.zip](https://github.com/07th-mod/ace-higurashi-attorney/files/8900809/FixAudioIssue.zip)

Make sure you extract it so the sound folder is in the same directory as the exe.
![image](https://user-images.githubusercontent.com/20390474/173607609-40da4ba9-6c34-4625-b28b-03b03d05805d.png)

Let me know if it works for you. If it's all good, we can package it and put together a proper release.

**Comment by ZXNova** [comment]
I extracted it and it seems to be working. Am hearing no sped up music

**Comment by ZXNova** [comment]
> I extracted it and it seems to be working. Am hearing no sped up music

Though the song during the first flashback seems to have a bad loop, it kinda clips when it loops

**Comment by DoctorDiablo** [comment]
Glad to hear it's working. Not sure if I have the audio knowhow to fix a bad loop, though.
